spring-boot-dependencies/pom.xml

```diff
@@ -273,6 +274,11 @@
 			</dependency>
 			<dependency>
 				<groupId>org.springframework.boot</groupId>
+				<artifactId>spring-boot-starter-data-couchbase</artifactId>
+				<version>1.4.0.BUILD-SNAPSHOT</version>
```

Comment:
Nope. What you're suggesting would be a nightmare IMO ;-) All Spring Boot starters have the same version: the version of Spring Boot you're using.